src/sparsezoo/objects/model_folder_objects.py

```diff
@@ -0,0 +1,442 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Class objects for standardization and validation of a model folder structure
+"""
+
+import json
+import logging
+import os
+import pathlib
+import re
+import tarfile
+from typing import List, Optional, Union
+
+import onnx
+
+from PIL import Image
+from src.sparsezoo.utils.numpy import load_numpy_list
```

Comment:
```suggestion
from sparsezoo.utils.numpy import load_numpy_list
```

Reply:
Adding it to the new PR in the other branch.